index.html

```diff
@@ -2968,26 +3000,47 @@ <h3>Data Schema Serialization</h3>
 </section> <!-- end of id="data-schema-serialization-json"-->
 
 <section id="titles-descriptions-serialization-json">
-<h3><code>title(s)</code> and <code>description(s)</code></h3>
+<h3>Human-Readable Metadata</h3>
 
   <p>
   The vocabulary terms <code>title</code> and <code>description</code> are 
   used to provide human-readable texts for titles (e.g., a display
   text for UI) and additional information useful for <a>Consumers</a> of TDs.
-  When a default language is specified by assigned a value to <code>@language</code>
+  When a default language is specified by assigning a value to <code>@language</code>
   in the <code>@context</code> of a <a href="#sec-thing-as-a-whole-json">Thing object</a>,
   that default language is associated with the string values assigned to
   <code>title</code> and <code>description</code> within the TD
-  and to <code>name</code> in the Thing object.
+  and to <code>name</code> and <code>support</code> in the Thing object.
+  </p>
+  <p>
   Another possibility is to set the default language with
-  the language negotiation mechanism of HTTP
-  (i.e., the <code>Accept-Language</code> header field).
+  a language negotiation mechanism, such as 
+  the <code>Accept-Language</code> header field of HTTP.
+  <span class="rfc2119-assertion" id="td-ns-multilanguage-content-negotiation">
+  In protocols such as HTTP that have an ability to negotiate content based on languages,
+  terms (e.g. <code>description</code> and <code>title</code>) intended
+  for carrying texts in the default language SHOULD be used instead of <code>MultiLanguage</code> container
+  when the texts are served as a result of such negotiation.</span>
+  Note however that constrained devices may choose to not support dynamically-generated TDs
+  and do not need to support this feature.
+  </p>
+  <p>
+  As noted earlier, the base text direction used to display human-readable text is also inferred 
+  from the default language, and so if a language can be written in more than
+  one script with different text directions, a script subtag needs to be 
+  included in the language tag.
+  <span class="rfc2119-assertion" id="td-context-default-language-direction-independence">
+  Each human-readable string value MUST be processed independently, using the base
+  direction defined for that string.</span>  In other words, a processor cannot
+  carry forward changes in direction from one string to another, or infer direction
+  for one string from another one elsewhere in the TD.
   </p>
-
   <p>
   A TD snippet using <code>title</code> and <code>description</code> is 
   shown below. The default language is set to <code>"en"</code> through the
```

Comment:
remove quotes around `en` in code tags.

Reply:
Actually, elsewhere you had me remove the "code" markup around language tags, too.   We should be consistent.   It looks a little odd with nothing around it, and it is a string value.   I think the other places language tags are given as examples they should get the "code" treatment to be consistent.

Reply:
OK, fixed... but I also went back and put the language tag examples in the "Multilanguage" section inside "code" markup to be consistent.

Reply:
Sorry, I cannot remember saying that the code markup should be removed from language tag examples. As they are Terms, they should have code markup.